website/src/graphql/authExchange.ts

```diff
@@ -0,0 +1,70 @@
+import { authExchange } from "@urql/exchange-auth"
+import { CognitoIdToken } from "amazon-cognito-identity-js"
+import { makeOperation } from "urql"
+import { getCurrentUser, getIdToken } from "src/auth"
+import {
+  GRAPHQL_URL_READ,
+  GRAPHQL_URL_WRITE,
+  WP_GRAPHQL_URL,
+} from "src/graphql"
+
+const secondsSinceEpoch = () => Date.now() / 1000
+
+const isExpired = (token: CognitoIdToken) =>
+  token.getExpiration() <= secondsSinceEpoch()
```

Comment:
I think this is right. The API has no docstring for `getExpiration`

Reply:
I'm following this which is based on the JWT spec https://stackoverflow.com/questions/39926104/what-format-is-the-exp-expiration-time-claim-in-a-jwt